prow/config/jobs/test-infra.yaml

```diff
@@ -136,3 +136,24 @@ jobs:
     requirements: [docker, gcp]
     node_selector:
       prod: prow
+
+  - name: bump-k8s-prow-images
+    type: periodic
+    cron: "05 15-23 * * 1-5"  # Run at 7:05-15:05 PST (15:05 UTC) Mon-Fri
+    command:
+    - ./tools/automator/automator.sh
+    - --org=istio
+    - --repo=test-infra
+    - "--title=Automator: bump k8s-prow images"
+    - --modifier=bump-k8s-prow-images
+    - --token-path=/etc/github-token/oauth
+    - --script-path=../test-infra/tools/automator/scripts/update-images.sh
+    - --verbose
+    - --
+    - --paths=prow/cluster/jobs/**/*.yaml,prow/config/jobs/**/*.yaml,.prow.yaml
```

Comment:
Doesn't this need to exclude `prow/cluster/jobs/istio/test-infra`?

Reply:
Technically no, since the image version of plank (i.e. --source) should always be the same as the image versions of jobs in `test-infra/` (since they are bumped together by autobump).

It doesn't hurt to exclude them though but I will need to enable `extglob` to ensure this remains simple.

Reply:
Ah so that should always be a noop. 👍 
Excluding might be preferable, but it doesn't seem important and shouldn't hold this up.

Reply:
I went ahead and added an exclusion to tidy this up.